src/test/java/com/spirnt/mission/discodeit/controllerTests/ChannelControllerTest.java

```diff
@@ -0,0 +1,258 @@
+package com.spirnt.mission.discodeit.controllerTests;
```

Comment:
보통 패키지명은 카멜케이스를 사용하지 않아서 소문자로 전부 표기하는게 좋을 것 같아요.  
```suggestion
package com.spirnt.mission.discodeit.controllertests;
```